### ISSUE TITLE
Misordered function args in keras.ops.normalize for symbolic tensors

### ISSUE THREAD

**Issue by sirfz** [keras-team/keras#19215]
I worked on porting a TF-specific model to keras 3 ops and stumbled over an issue with `keras.ops.normalize` (or `keras.utils.normalize`):

When passing a symbolic tensor as input (e.g. `keras.utils.normalize(symbolic, axis=1, order=2)`), it returns `Normalize(axis, order).symbolic_call(symbolic)`. However, in `Normalize` the `call` method calls `_normalize(x, self.order, self.axis)` instead of `_normalize(x, self.axis, self.order)` (reference: https://github.com/keras-team/keras/blob/e6e62405fa1b4444102601636d871610d91e5783/keras/ops/nn.py#L1855).

The fix is to simply swap `self.axis` and `self.order` (or use keyword arguments instead?). I can submit a PR.

**Comment by fchollet** [comment]
Thanks for the report -- nice catch. I just fixed it.
